try {
    // some code
}
catch (a: unknown) {
    // handle error
}


try {
    // some code
}
catch (a) {
    // handle error
}